controllers/books.mjs

```diff
@@ -1,4 +1,6 @@
 import { BooksStore as books } from '../models/books-store.mjs';
+import { default as summaryDoc } from '../entities/pdf-summary.mjs';
+import { approotdir } from '../approotdir.mjs';
```

Comment:
just as a comment,  usually by convention a line break is left after import/requre
```suggestion
import { approotdir } from '../approotdir.mjs';

```

Reply:
Sure, thanks :smile: